### PR TITLE
Fix tests after v5 -> update-to-v5

### PR DESCRIPTION
**Pull request by EsoterikStare**
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/mui-org/material-ui/blob/HEAD/CONTRIBUTING.md#sending-a-pull-request).

### CODE REVIEW

File: packages/material-ui/src/Menu/Menu.js [hunk 1]
```diff
@@ -149,18 +149,19 @@ const Menu = React.forwardRef(function Menu(inProps, ref) {
     event.preventDefault();
     setOpenSubMenuIndex(null);
     if (onClose) {
-      onClose(event, `${(event && event.key && event.key.toLowerCase()) || 'unknown'}KeyDown`);
+      onClose(event, `${event.key.toLowerCase()}KeyDown`);
```
Comment (kingedward35) [inline packages/material-ui/src/Menu/Menu.js:152]: Why was this changed?

File: packages/material-ui/src/Menu/Menu.js [hunk 2]
```diff
@@ -149,18 +149,19 @@ const Menu = React.forwardRef(function Menu(inProps, ref) {
     event.preventDefault();
     setOpenSubMenuIndex(null);
     if (onClose) {
-      onClose(event, `${(event && event.key && event.key.toLowerCase()) || 'unknown'}KeyDown`);
```
Comment (kingedward35) [inline packages/material-ui/src/Menu/Menu.js:152]: Why was this changed?

Comment (EsoterikStare) [reply packages/material-ui/src/Menu/Menu.js:152]: Because I found it to be unnecessary now and it was also a little weird with that fallback case because you could get an event name of `unknownKeyDown` which is definitely not a MUI thing. It seems pretty safe since this only fires during a keydown event which should always have an event with a key on it to `toLowerCase()`.

Comment (kingedward35) [reply packages/material-ui/src/Menu/Menu.js:152]: Cool.
